test/core/slice/b64_test.cc

```diff
@@ -120,101 +120,90 @@ static void test_url_safe_unsafe_mismatch_failure(void) {
   grpc_core::ExecCtx exec_ctx;
   b64 = grpc_base64_encode(orig, sizeof(orig), url_safe, 0);
   orig_decoded = grpc_base64_decode(b64, !url_safe);
-  GPR_ASSERT(GRPC_SLICE_IS_EMPTY(orig_decoded));
+  ASSERT_TRUE(GRPC_SLICE_IS_EMPTY(orig_decoded));
   gpr_free(b64);
   grpc_slice_unref_internal(orig_decoded);
 
   b64 = grpc_base64_encode(orig, sizeof(orig), !url_safe, 0);
   orig_decoded = grpc_base64_decode(b64, url_safe);
-  GPR_ASSERT(GRPC_SLICE_IS_EMPTY(orig_decoded));
+  ASSERT_TRUE(GRPC_SLICE_IS_EMPTY(orig_decoded));
   gpr_free(b64);
   grpc_slice_unref_internal(orig_decoded);
 }
 
-static void test_rfc4648_test_vectors(void) {
+TEST(B64Test, Rfc4648TestVectors) {
   char* b64;
 
   b64 = grpc_base64_encode("", 0, 0, 0);
-  GPR_ASSERT(strcmp("", b64) == 0);
+  ASSERT_STREQ("", b64);
   gpr_free(b64);
 
   b64 = grpc_base64_encode("f", 1, 0, 0);
-  GPR_ASSERT(strcmp("Zg==", b64) == 0);
+  ASSERT_STREQ("Zg==", b64);
   gpr_free(b64);
 
   b64 = grpc_base64_encode("fo", 2, 0, 0);
-  GPR_ASSERT(strcmp("Zm8=", b64) == 0);
+  ASSERT_STREQ("Zm8=", b64);
   gpr_free(b64);
 
   b64 = grpc_base64_encode("foo", 3, 0, 0);
-  GPR_ASSERT(strcmp("Zm9v", b64) == 0);
+  ASSERT_STREQ("Zm9v", b64);
   gpr_free(b64);
 
   b64 = grpc_base64_encode("foob", 4, 0, 0);
-  GPR_ASSERT(strcmp("Zm9vYg==", b64) == 0);
+  ASSERT_STREQ("Zm9vYg==", b64);
   gpr_free(b64);
 
   b64 = grpc_base64_encode("fooba", 5, 0, 0);
-  GPR_ASSERT(strcmp("Zm9vYmE=", b64) == 0);
+  ASSERT_STREQ("Zm9vYmE=", b64);
   gpr_free(b64);
 
   b64 = grpc_base64_encode("foobar", 6, 0, 0);
-  GPR_ASSERT(strcmp("Zm9vYmFy", b64) == 0);
+  ASSERT_STREQ("Zm9vYmFy", b64);
   gpr_free(b64);
 }
 
-static void test_unpadded_decode(void) {
+TEST(B64Test, UnpaddedDecode) {
   grpc_slice decoded;
 
   grpc_core::ExecCtx exec_ctx;
   decoded = grpc_base64_decode("Zm9vYmFy", 0);
-  GPR_ASSERT(!GRPC_SLICE_IS_EMPTY(decoded));
-  GPR_ASSERT(grpc_slice_str_cmp(decoded, "foobar") == 0);
+  ASSERT_FALSE(GRPC_SLICE_IS_EMPTY(decoded));
```

Comment:
this is now a redundant assert (same throughout)

Reply:
done.